docs/templates/properties.js

```diff
@@ -39,6 +39,20 @@ module.exports['has-errors'] = function ({ item }) {
   return item.inheritance.some(c => c.errors.length > 0);
 };
 
+module.exports.functions = function ({ item }) {
+  return [...findAll(['VariableDeclaration', 'FunctionDefinition'], item)].filter(f =>
+    f.nodeType === 'VariableDeclaration' ? f.visibility === 'public' : f.visibility !== 'private',
+  );
+};
```

Comment:
IIUC this should be updated in [solidity-docgen](https://github.com/OpenZeppelin/solidity-docgen), is it?

Reply:
Yes, but it's more complicated because of backwards compatibility. `functions` currently returns `FunctionDefinition` nodes and not `VariableDeclaration` nodes, so it might break some templates.